lua/map_scripts/maps/gm_bigcity_winter_day.lua

```diff
@@ -0,0 +1,8 @@
+local scareButtonNumber = 3342
+
+-- Removes the mapwide jumpscare sound
+hook.Add( "InitPostEntity", "CFC_MapScripts_bigcitywinter", function()
+    local ent = ents.GetMapCreatedEntity( scareButtonNumber )
+    if not IsValid( ent ) then return end
```

Comment:
Just in case it changes, this'll alert us instead of just randomly finding out players are abusing it in-game, lol

```suggestion
    if not IsValid( ent ) then
        return ErrorNoHaltWithStack( "Couldn't find jumpscare button!" )
    end
```